utils/scripts/ocm/ocm.py

```diff
@@ -639,6 +638,17 @@ def wait_for_osd_cluster_to_get_deleted(self, timeout=3600):
                    " EXITING".format(self.cluster_name))
             sys.exit(1)
 
+    def install_rhoda_addon(self):
+        if not self.is_addon_installed("dbaas-operator"):
+            self.install_addon("dbaas-operator")
+            self.wait_for_addon_installation_to_complete("dbaas-operator")
+        # Waiting 5 minutes to ensure all the services are up
+        time.sleep(300)
```

Comment:
Can we avoid blind sleep? Like we are waiting for the installation to complete, can we create a method that would ensure the required services are in running state?

Reply:
Okay, I agree with the go forward plan. LGTM